.eslintrc.json

```diff
@@ -1,15 +1,27 @@
 {
   "root": true,
   "extends": [
-    "react-app",
+    "airbnb",
+    "airbnb-typescript",
+    "airbnb/hooks",
     "plugin:prettier/recommended"
   ],
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
   "ignorePatterns": [
     // node_modules is implicitly always ignored
-    "dist"
+    "dist",
+    "vite.config.mts",
+    "jest.config.ts",
+    "jest.setup.ts"
```

Comment:
Why?

Reply:
There was an error during build because these files are not included in `tsconfig.json`, I created a separate tsconfig for eslint to keep the main `tsconfig.json` unchanged

Reply:
Finally, I kept it as it is, so these config and setup files would be ignored